libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/Microsoft.Bot.Builder.Adapters.Facebook.csproj

```diff
@@ -5,7 +5,7 @@
     <PackageVersion Condition=" '$(ReleasePackageVersion)' == '' ">$(LocalPackageVersion)</PackageVersion>
     <PackageVersion Condition=" '$(ReleasePackageVersion)' != '' ">$(ReleasePackageVersion)</PackageVersion>
     <Configurations>Debug;Release</Configurations>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild>$(GeneratePackages)</GeneratePackageOnBuild>
```

Comment:
Can we just delete this from all 30 projects in the libraries and add it to the https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Directory.Build.props?
![image](https://user-images.githubusercontent.com/12264946/97627671-120c8800-1a02-11eb-80c0-bca5d531ae1d.png)

Reply:
I tried that. Then when you want to create packages, the projects that cannot create packages throw errors. (We don't want a package for every project.)

Reply:
Excellent idea. Done.